src/index.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable prefer-template */
+/* eslint-disable camelcase */
+import './style.css';
+import './style_mobile.css';
+import Icon from './assest/logo1.png';
+import getData from './modules/homePage.js';
+import Likes from './modules/Likes.js';
+import CalculeLike from './modules/CalculeLike.js';
+import handleForm, { getCommentData } from './modules/comment.js';
+
+const mylogo = new Image();
+mylogo.src = Icon;
+const logo = document.querySelector('.logo');
+logo.appendChild(mylogo);
+
+const container = document.querySelector('.meals_container');
+
+const display = async () => {
+  const meals = await getData('https://www.themealdb.com/api/json/v1/1/search.php?f=a');
+  let liked = [];
+  if (localStorage.getItem('liked') != null) {
+    const lik = JSON.parse(localStorage.getItem('liked'));
+    liked = lik.map((item) => item.item_id);
+  }
+
```

Comment:
- [ ] I noticed that when I click the comment button and complete a comment, everything works properly but I noticed that if I exit the popup and try to click the comment button, I get a blank screen until I refresh the page before I see contents. Kindly fix it and make your application work fine. I recorded a gif to explain the bug in the application.

![chrome-capture (2)](https://user-images.githubusercontent.com/77106962/163683250-78ac0303-3393-46cb-8c2d-acf5227cdb7a.gif)

Reply:
Thank you 👍 I resolve it